go/vt/schemadiff/errors.go

```diff
@@ -389,11 +389,20 @@ func (e *IndexNeededByForeignKeyError) Error() string {
 }
 
 type ViewDependencyUnresolvedError struct {
-	View string
+	View                      string
+	MissingReferencedEntities []string
 }
 
 func (e *ViewDependencyUnresolvedError) Error() string {
-	return fmt.Sprintf("view %s has unresolved/loop dependencies", sqlescape.EscapeID(e.View))
+	var b strings.Builder
+	b.WriteString(fmt.Sprintf("view %s has unresolved/loop dependencies: ", sqlescape.EscapeID(e.View)))
```

Comment:
Can this use `fmt.Fprintf` with the buffer as the `io.Writer`?

Reply:
Done this :smile:.

Reply:
nice